src/Makefile.in

```diff
@@ -155,7 +155,7 @@ install:	install-headers install-libs
 install-python:
 	cd python && python setup.py install
 
-tests: test/histogram_test test/histogram_perf $(LUA_FFI)
+tests: test/histogram_test test/histogram_perf $(LUA_FFI) $(PYTHON_FFI)
 	test/runTest.sh
```

Comment:
This change will make CI (ours and Travis) dependent on python being setup correctly.
If we are willing to do that, we should also run the python tests here.

Reply:
Looks like we are fine with this. So 👍